.github/workflows/docker.yml

```diff
@@ -0,0 +1,22 @@
+name: Create and publish Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  call_build-and-push-image:
+    permissions:
+      contents: read
+      packages: write
+    uses: eurofurence/reg-workflows/.github/workflows/docker-build-push.yml@main
+    with:
+      image-name: ${{ github.repository }}
+      image-tags: latest
```

Comment:
Since we seem to be planning on running this only on `main` for now anyways, we might want to consider providing at least the current version or short SHA (`git rev-parse --short HEAD`) as additional tag so that previous releases remain available in GHCR e.g. to allow for rollbacks of deployments.

Reply:
Not yet, but this is planned once we set versions. Right now I just need the latest images to start building a helm chart.